apps/services/auth-public-api/src/app/modules/delegations/delegations.controller.ts

```diff
@@ -0,0 +1,87 @@
+import {
+  DelegationsService,
+  IDelegation,
+  DelegationDTO,
+  Delegation,
+} from '@island.is/auth-api-lib'
+import {
+  IdsUserGuard,
+  Scopes,
+  ScopesGuard,
+  User,
+  CurrentUser,
+} from '@island.is/auth-nest-tools'
+import {
+  Body,
+  Controller,
+  Delete,
+  Get,
+  Param,
+  Post,
+  Put,
+  UseGuards,
+} from '@nestjs/common'
+import { ApiCreatedResponse, ApiOkResponse, ApiTags } from '@nestjs/swagger'
+
+@UseGuards(IdsUserGuard, ScopesGuard)
+@ApiTags('delegations')
+@Controller('delegations')
+export class DelegationsController {
+  constructor(private readonly delegationsService: DelegationsService) {}
+
+  @Scopes('@island.is/auth/delegations:read')
+  @Get()
+  @ApiOkResponse({ isArray: true })
+  async findAllTo(@CurrentUser() user: User): Promise<IDelegation[]> {
+    return await this.delegationsService.findAllTo(user.nationalId)
+  }
```

Comment:
This is an endpoint that we want to be able to call with the currently "authenticated" user rather than the delegated user, i.e. `user.act?.nationalId ?? user.nationalId`.

We probably want to define an API naming convention for that (/me/delegations vs /delegations or /subject/delegations? Or just /users/:nationalId/delegations?) as well as nest helpers.

Reply:
cc @valurefugl

Reply:
async findAllTo(@CurrentActor() user: User) ?

Reply:
https://github.com/island-is/identity-server.web/issues/275

Reply:
Yeah, CurrentActor LGTM.